lib/Test/Script.pm

```diff
@@ -564,7 +766,7 @@ BEGIN {
 =head1 CAVEATS
 
 This module is fully supported back to Perl 5.8.1.  In the near future, support
-for the older pre-Test2 Test::Builer will be dropped.
+for the older pre-Test2 Test::Builder will be dropped.
```

Comment:
This is out of date.  We already dropped the older version of `Test::Builder` in favor of `Test2::API`, if you don't mind can you remove the second sentence?